install.ps1

```diff
@@ -0,0 +1,8 @@
+# Create a virtual environment
+python3 -m venv .venv
+
+# Activate the virtual environment
+& .\.venv\Scripts\Activate.ps1
+
+# Install the package in the virtual environment
+pip install .
```

Comment:
Windows? Did this work on windows? Curious if it works there.

Reply:
I was testing on Windows but couldn't fully get it working right. Will test again and make sure as I switched to using Linux to do further dev

Reply:
@AlexCheema I can confirm that it is working on Windows, but there are a few issues:

1) PyTorch 2.4 doesn't install. In order to get it working, it needs the nightly build

2) In main.py

```
for s in [signal.SIGINT, signal.SIGTERM]:
      loop.add_signal_handler(s, handle_exit)
```
isn't supported on windows. 

If I change `handle_exit()` to:

```
def handle_exit():
    asyncio.ensure_future(shutdown(loop))

  if platform.system() != "Windows":
    for s in [signal.SIGINT, signal.SIGTERM]:
      loop.add_signal_handler(s, handle_exit)
  else:
    # On Windows, we can only reliably catch SIGINT (Ctrl+C)
    signal.signal(signal.SIGINT, lambda signum, frame: handle_exit())
```
3) Getting some kind of network error between the GUI and the backend

It seems to work. It's insanely slow for me though (no GPU... the raspberry pis are much faster 😄). Windows changes perhaps out of scope for this PR though.

![image](https://github.com/user-attachments/assets/5bb0176f-5cd8-4043-8610-fd7c46757c2e)